### DISCUSSION TITLE
Implement page patch and other api endpoints

### DISCUSSION THREAD

**Discussion by markacola** [jakeswenson/notion#54] [Ideas]
It would be great to extend the current set to include all the general notion API endpoints; the most notable for my use would be [the page patch endpoint](https://developers.notion.com/reference/patch-page).

Perhaps creating/updating the existing Roadmap project with a burndown of all the available API endpoints.

Happy to contribute to creating the roadmap and/or adding functionality; let me know if extending is the intent of the lib.

**Comment by plule** [comment]
Hello, interested in page update too! I've looked around it looks like someone actually already started working on that [here](https://github.com/yancouto/notion). @yancouto is this in a working state already by any chance?

**Comment by yancouto** [comment]
I've used it for a pet project and it works :), but haven't touched it since then
